src/ORM/DataList.php

```diff
@@ -1344,36 +1334,30 @@ public function offsetExists($key)
      *
      * The object returned is not cached, unlike {@link DataObject::get_one()}
      *
-     * @param mixed $key
      * @return DataObject
      */
-    #[\ReturnTypeWillChange]
-    public function offsetGet($key)
+    public function offsetGet($key): mixed
```

Comment:
```suggestion
    public function offsetGet($key): DataObject
```
This can't return anything _other_ than a `DataObject` can it? Or maybe `?DataObject`

Reply:
Fixed.